pkgs/top-level/all-packages.nix

```diff
@@ -30874,15 +30874,23 @@ with pkgs;
 
   bleachbit = callPackage ../applications/misc/bleachbit { };
 
-  blender = callPackage  ../applications/misc/blender {
-    # LLVM 11 crashes when compiling GHOST_SystemCocoa.mm
-    stdenv = if stdenv.isDarwin then llvmPackages_10.stdenv else stdenv;
-    inherit (darwin.apple_sdk.frameworks) Cocoa CoreGraphics ForceFeedback OpenAL OpenGL;
-  };
+  inherit (callPackages ../applications/misc/blender { }) 
+    blender3_3
+    blender3_6
+    blender
+    blender-lts
+  ;
+
+  inherit (callPackages ../applications/misc/blender { hipSupport = true; })
+    blender3_3-hip
```

Comment:
If I'm not mistaken, this pattern breaks `blender.override`.

Reply:
Changed to `inherit (recurseIntoAttrs`, would that do it? I don't know how to test for it.

Reply:
To test, just try `nix-instantiate --expr 'with import <nixpkgs> {}; blender.override { inherit stdenv; }'`

Reply:
> No. There would need to be separate callPackages for each attribute.

Okay I've tested it, overrides should work. `callPackages` are created with each attribute in `default.nix`